__tests__/setup/web/index.html

```diff
@@ -138,7 +157,24 @@
           };
         });
 
+        const activityMiddleware = () => next => card => {
```

Comment:
Why are you using the activity middleware instead of the store middleware?

Reply:
Because we can only run the hook while we are doing rendering.